docs/GettingStarted.rst

```diff
@@ -1432,23 +1436,73 @@ News application filters
 
 Different filters are available from News application so you can easily find articles posted in specific spaces, pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.
 
-You can search articles by spaces using the filter button from the top bar.
+
+**Filter articles per spaces : ** You can search articles by spaces using the filter button from the top bar.
 
 |image206|
 
-In the displayed drawer, you can select the spaces of the articles you want to filter. A search input is available to easily find the spaces you want to select. Only the spaces you are member of are available. A click on the Apply button allows to apply the filter.
+In the displayed drawer, you can choose from or search in the list of the spaces in which you are member the space(s) you want to filter by articles in the application then apply the using the button to be redirected to the filtered view. 
```

Comment:
I think you revert the changes we discussed in the previous PR. Is it intended ?

Reply:
I didn't change this part. I created a new branch from master after doing a git fetch git pull. Will check

Reply:
Fixed